src/components/Notification/InactivityMessage.jsx

```diff
@@ -14,15 +16,13 @@ import { LogoutButton } from '@inrupt/solid-ui-react';
  */
 
 const InactivityMessage = () => {
-  const navigate = useNavigate();
-
   const [showPopup, setShowPopup] = useState(false);
   const [activeUser, setActiveUser] = useState(false);
 
   // Checks for active user by looking for a loggedIn key in local storage
   useEffect(() => {
     const activeCheck = localStorage.getItem('loggedIn');
-    if (activeCheck) setActiveUser(activeCheck);
+    if (activeCheck) setActiveUser(true);
```

Comment:
I'm not a big fan of unnecessary branching. Could we do
 `setActiveUser(activeCheck == true)` or `setActiveUser(activeCheck !== null)`

Reply:
Yeah, I think we can use setActiveUser(activeCheck == true) on Line 25 to evaluate the condition inline.

Reply:
I've used the first expression suggested as a solution to the if branching, but I had to modify it. ESLint seems to take issue for `==`. The linter was expecting `===`.

<img width="606" alt="Screen Shot 2023-05-15 at 12 43 42" src="https://github.com/codeforpdx/PASS/assets/14917816/e97bdb65-c094-4590-8cc4-b76cb5ee18aa">

It make sense to use `==` to evaluate our expression since loggedIn suppose to come out as a string from getItem, but I don't want to overwrite what we have set up right now for lints.

I'm okay with using Boolean(activeCheck) if we want to keep the boolean `true` (solution 1: `Boolean(activeCheck) === true`), but I think using the string `'true'` would be more concise (solution 2: `activeCheck === 'true'`).